docs/configuration.md

```diff
@@ -0,0 +1,212 @@
+# Configuration
+
+## terra.yaml
+
+The terra.yaml file is the only Terra-specific addition required. This file defines inputs that the web 
+UI uses to display storage options, connection settings, and configuration parameters.
```

Comment:
again, I'd link to the genesis docs. so users know where this UI is.

Reply:
These docs aren't available yet. We can make a note for when they are ready.